src/aioice/ice.py

```diff
@@ -486,7 +486,9 @@ async def add_remote_candidate(self, remote_candidate: Optional[Candidate]) -> N
                 self._check_list.append(pair)
         self.sort_check_list()
 
-    async def gather_candidates(self) -> None:
+    async def gather_candidates(
+        self, error_handler: Optional[Callable[[Exception], None]] = None
```

Comment:
Sorry this change was contaminated from my local trials to make the parent RTCPeerConnection to dispatch [onicecandidateerror event](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/icecandidateerror_event) at each ICE gathering error but it shouldn't be in this PR.
I will revert it.